prep-html-links/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Home Page</title>
+</head>
+<body>
+  <h1>Home page</h1>
+  <a href="file:///C:/Users/kirei/repos/prep-0721-code-solutions/prep-html-links/contact-us/contact-us.html">Click here to go to contact page</a>
```

Comment:
Rather than use the absolute file path to the `contact-us.html`, use a relative path to the `contact-us.html` inside of the nearby `contact-us/` directory (_i.e._ `href="contact-us/contact-us.html"`).

That way, the link works on computers other than your own.

Reply:
![html links v 2](https://user-images.githubusercontent.com/88108910/128090875-f4b00fb6-f6df-49f5-a70d-739e96d43626.png)
![index link v 2](https://user-images.githubusercontent.com/88108910/128090877-eb05e765-9e2f-4b6d-baad-6581bca01e44.png)
![html link 2](https://user-images.githubusercontent.com/88108910/128090881-4c83b9fa-59df-4e1c-bace-a1355823abba.gif)